pkg/graph/schema.resolvers_helpers.go

```diff
@@ -51,9 +51,9 @@ func ConvertToPlanBasics(mpi *model.PlanBasicsInput) *models.PlanBasics {
 	if mpi.ID != nil {
 		basics.ID = *mpi.ID
 	}
-	if mpi.Status != nil {
-		basics.Status = *mpi.Status
-	}
+	// if mpi.Status != nil {
+	// 	basics.Status = *mpi.Status
+	// }
```

Comment:
Is this intended to be commented out? Seems like with this change we'd always have a `nil` `basics.Status`

Reply:
yeah, I'd agrees with this, it should be commented out